src/collection.rs

```diff
@@ -308,13 +311,25 @@ impl<'a, C: ClientExt> Collection<'a, C> {
         let resp: CollectionChecksum = serialize_response(self.session.get(url, "").await?.text())?;
         Ok(resp)
     }
-    /// Loads a collection into memory.
+
+    /// Loads a collection into memory. Returns the collection on success.
     ///
+    /// The request body object might optionally contain the following attribute:
+    /// - count: If set, this controls whether the return value should include the number of documents in the collection.
+    /// Setting count to false may speed up loading a collection. The default value for count is true.
     /// # Note
     /// this function would make a request to arango server.
     #[maybe_async]
-    pub async fn load(&self) {
-        unimplemented!()
+    pub async fn load(&self, count: bool) -> Result<CollectionLoad, ClientError> {
+        let url = self.base_url.join(&format!("load")).unwrap();
```

Comment:
Here `Url::join` accepts a `&str`, and `"load"` is already a `&str`. So we don't need to allocate memory to store "load" as a `String` and convert it to `&str`.

The following code suffices:
```rust
let url = self.base_url.join("load").unwrap();
```

Reply:
Ho yes ! True ! Thank you, I' ll make the update.